series/liberec-pyvo/events/2018-02-14.yaml

```diff
@@ -28,11 +28,13 @@ talks:
       jak za 10 minut vytvořit nádherné *CUI* (command-line user interface)
       s pomocí modulu `click`. Pokročilé parsování argumentů je jenom
       začátek – je libo barvičky nebo progressbar?
+    coverage:
+      - video: https://www.youtube.com/watch?v=kRY2UGhjRb0
```

Comment:
Je tohle ok? Dát video na přednášku odjinud (přestože stejný člověk a asi i stejná přednáška?). Já nedokážu rozhodnout, prosím @encukou.

Reply:
Jo, nad tím jsem taky přemýšlel. Nadpis té sekce je „Jinde na webu“, z toho mi vyplývá, že to není problém.
Na druhou stranu, web pyvo.cz bez ohledu na typ odkazu (jestli to je link, slidy, nebo cokoli jiného), když v tom uvidí YouTube video id, tak z toho udělá náhled. Ale počítám s tím, že až/jestli se povede vyrobit opravdový záznam, tak se umístí nad to a tím pádem vyhraje.

Reply:
To bych nedělal. Není to ani nahrávka z Pyva, byl by v tom zmatek.
Co takhle dát odkaz jen do popisku?

Reply:
OK, opraveno.

Reply:
Díky!